docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -98,17 +98,146 @@ These templates are loaded when the integration is installed, and are used to co
 [[data-streams-ilm]]
 == Configure an index lifecycle management (ILM) policy
 
-Use the {ref}/getting-started-index-lifecycle-management.html[index lifecycle
+Use the {ref}/index-lifecycle-management.html[index lifecycle
 management] (ILM) feature in {es} to manage your {agent} data stream indices as they age.
 For example, create a new index after a certain period of time,
 or delete stale indices to enforce data retention standards.
 
-{agent} uses ILM policies built-in to {es} to manage backing indices for its data streams.
-See the {ref}/example-using-index-lifecycle-policy.html[Customize built-in ILM policies] tutorial
-to learn how to customize these policies based on your performance, resilience, and retention requirements.
+Installed integrations may have one or many associated data streams--each with an associated ILM policy.
+By default, these data streams use an ILM policy that matches their data type.
+For example, the data stream `metrics-system.logs-*`,
+uses the metrics ILM policy as defined in the `metrics-system.logs` index template.
 
-To instead create a new ILM policy, in {kib},
-go to **Stack Management** > **Index Lifecycle Policies**. Click **Create policy**.
-Define data tiers for your data, and any associated actions,
-like a rollover, freeze, or shrink.
-See {ref}/set-up-lifecycle-policy.html[configure a lifecycle policy] for more information.
+[discrete]
+[[data-streams-ilm-tutorial]]
+== Tutorial: Customize data retention for integrations
+
+This tutorial explains how to apply a custom ILM policy to an integration's data stream.
+
+**Scenario:** You have {agent}s collecting system metrics with the System integration in two environments--
+one with the namespace `development`, and one with `production`.
+
+**Goal:** Customize the ILM policy for the `system.network` data stream with the `production` namespace.
```

Comment:
This sentence doesn't parse right for me. Did you mean to say "within" instead of "in" here?

Reply:
```suggestion
**Goal:** Customize the ILM policy for the `system.network` data stream in the `production` namespace.
```